CHANGELOG.md

```diff
@@ -1,11 +1,24 @@
 Changelog
 =========
 
+1.7.x (Unreleased)
```

Comment:
Because new features are added (as I see in the section below) this should be 1.8.0